ibm/mas_devops/roles/suite_app_install/vars/defaultspecs/predict.yml

```diff
@@ -2,4 +2,9 @@
 # Default application spec for Predict
 mas_app_spec:
   bindings:
+    health: workspace
     jdbc: system
+    monitor: workspace
+    watsonstudio: system
+  components: {}
+  displayName: Predict Workspace
```

Comment:
Linting failure as there's no empty line, the build needs to pass before this can be merged in